auditors/nonroot/nonroot.go

```diff
@@ -59,16 +98,12 @@ func auditContainer(container *k8stypes.ContainerV1, resource k8stypes.Resource)
 		}
 	}
 
-	podSpec := k8s.GetPodSpec(resource)
-	if podSpec == nil {
-		return nil
-	}
 	if isContainerRunAsNonRootNil(container) {
 		if isPodRunAsNonRootNil(podSpec) {
 			return &kubeaudit.AuditResult{
 				Name:     RunAsNonRootPSCNilCSCNil,
 				Severity: kubeaudit.Error,
-				Message:  "runAsNonRoot is not set in container SecurityContext nor the PodSecurityContext. It should be set to 'true' in at least one of the two.",
+				Message:  "neither runAsNonRoot nor runAsUser are set in container SecurityContext or in the PodSecurityContext. runAsNonRoot should be set to 'true' or runAsUser should be set to a non-root UID either in container SecurityContext or PodSecurityContext.",
```

Comment:
```suggestion
				Message:  "runAsNonRoot should be set to 'true' or runAsUser should be set to a non-root UID either in the container SecurityContext or PodSecurityContext.",
```

Reply:
I think it would be a bit more succinct/clearer if we make the message a bit shorter. What do you think?

Reply:
Indeed better. Done.